src/cli.js

```diff
@@ -86,6 +86,7 @@ commander
   .command("start")
   .alias("s")
   .description("start everything")
+  .option("-M, --entrypoints <entrypoints>", "Enter specific")
```

Comment:
Let's take back `-E` for this.

Do we have to have a single letter? I don't think `--log-pretty` needs a single letter entry. If it must have a single letter then maybe we just call it `Z` for `--log-pretty`